packages/react-core/src/operations/constants/GroupDateTypes.d.ts

```diff
@@ -1,8 +1,8 @@
 export enum GroupDateTypes {
-  YEARS = 'years',
-  MONTHS = 'months',
-  WEEKS = 'weeks',
-  DAYS = 'days',
-  HOURS = 'hours',
-  MINUTES = 'minutes'
+  YEARS = 'year',
```

Comment:
Out of curiosity, why this change?

Reply:
Because in all the providers, these operations are singular :(